.github/workflows/update-browser-version.yml

```diff
@@ -30,3 +27,8 @@ jobs:
           echo "$browser_spec" > .browser
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v5
+        with:
```

Comment:
Set `delete-branch: true`?

> Delete the branch when closing pull requests, and when undeleted after merging. 

Default is false

Reply:
Should be okay since merging a PR deletes the branch automatically. Let's iterate on this detail later.